src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -2025,7 +1852,7 @@ private void CalculateEnergyAndCompoundBalance(IReadOnlyList<OrganelleTemplate>
     {
         biome ??= Editor.CurrentPatch.Biome;
 
-        bool moving = calculateBalancesWhenMoving.ButtonPressed;
+        bool moving = organismStatisticsPanel.CalculateBalancesWhenMoving();
```

Comment:
Shouldn't this be a property in the statistics? Pretty weird that a method with an imperative name would just return the status of something...